samples/redirects/1.a.getting-started-full-bundle/README.md

```diff
@@ -0,0 +1,7 @@
+# Redirect information
```

Comment:
This `README.md` should lives in /samples/1.a.getting-started-full-bundle because it need to be accessible to https://github.com/Microsoft/BotFramework-WebChat/samples/1.a.getting-started-full-bundle/ (for NPM README.md).

Reply:
🤦‍♀️ let me get that fixed